### ISSUE TITLE
App Store 

### ISSUE THREAD

**Issue by peteostro** [mikeswanson/SpatialPlayer#1]
Any plans to release this to the App Store?

**Comment by mikeswanson** [comment]
No, I don't plan to make any significant updates to this project, and I have no plans to release it to the App Store. I am aware of others who have used this code in their own projects, though, and it's my hope that they will.
